### ISSUE TITLE
supportedViewports missing for APL in the skill.json validation

### ISSUE THREAD

**Issue by kgnuuk** [alexa/ask-toolkit-for-vscode#10]
**Description:**

**Steps to reproduce the issue:**
1. Edit the skill.json
2. Add the interface 
```
{
            "type": "ALEXA_PRESENTATION_APL",
            "supportedViewports": [
              {
                "mode": "HUB",
                "shape": "ROUND",
                "minWidth": 480,
                "maxWidth": 480,
                "minHeight": 480,
                "maxHeight": 480
              },
              {
                "mode": "HUB",
                "shape": "RECTANGLE",
                "minWidth": 1024,
                "maxWidth": 1024,
                "minHeight": 600,
                "maxHeight": 600
              },
              {
                "mode": "HUB",
                "shape": "RECTANGLE",
                "minWidth": 1280,
                "maxWidth": 1280,
                "minHeight": 800,
                "maxHeight": 800
              },
              {
                "mode": "TV",
                "shape": "RECTANGLE",
                "minWidth": 960,
                "maxWidth": 960,
                "minHeight": 540,
                "maxHeight": 540
              },
              {
                "mode": "HUB",
                "shape": "RECTANGLE",
                "minWidth": 960,
                "maxWidth": 960,
                "minHeight": 480,
                "maxHeight": 480
              }
            ]
          }
```
**Observed result:**
Warning: Property supportedViewports is not allowed.

**Expected result:**
No Warning because supportedViewports is required to add APL support to all Alexa devices since it is possible to add APL for single devices

**Additional environment details (Ex: Windows, Mac, Amazon Linux etc)**
Mac
**VS Code version**:
1.38.1

**Comment by ShenChen-Amazon** [comment]
Hi @kgnuuk 
Thanks for reporting this issue. Its a known json validation issue and We are currently working on it. 
We inform you once we got it fixed. 

Thanks,
Shen

**Comment by ShenChen-Amazon** [comment]
Hi @kgnuuk 

We just published v1.0.2 and this issue should be fixed after upgrading the extension. Will close it for now and please feel free to reopen if you meet any problems.

Thanks,
Shen
